ui/src/main/webapp/app/query/operation-chain-component.js

```diff
@@ -108,14 +107,23 @@ function OperationChainController(operationChain, config, loading, query, error,
             chain.operations.push(operationService.createDeduplicateOperation(options));
         }
 
-        
         previousQueries.addQuery({
             name: "Operation Chain",
             lastRun: moment().format('HH:mm'),
             operations: vm.operations
         });
 
         runQuery(chain.operations);
+
+        var clearChainCheckboxState = settings.getClearChainCheckbox();
+        if (clearChainCheckboxState == true) {
+            vm.resetChainWithoutDialog()
+        }
+    }
+
+    vm.resetChainWithoutDialog = function() {
```

Comment:
Think the could be a private method (by using `var resetChainWithoutDialog = function() {` as opposed to `vm.resetChainWithoutDialog = function() {`.